tests/Unit/API/Google/AdsTest.php

```diff
@@ -282,4 +282,45 @@ public function test_update_billing_url() {
 			->willReturn( true );
 		$this->assertTrue( $this->ads->update_billing_url( self::TEST_BILLING_URL ) );
 	}
+
+	public function test_get_enhanced_conversion_status() {
+		$value = $this->options->expects( $this->once() )
+		->method( 'get' )
+		->with( OptionsInterface::ENHANCED_CONVERSION_STATUS )
+		->willReturn( 'enabled' );
+
+		$this->assertEquals( 'enabled', $this->ads->get_enhanced_conversion_status() );
+	}
+
+	public function test_update_enhanced_conversion_status_with_invalid_status_throws_exception() {
+		$this->expectException( Exception::class );
+		$this->expectExceptionMessage('Invalid state for enhanced conversion');
```

Comment:
These types of PHPCS issues should be fixable with phpcbf. `npm run lint:fix:php`.

Reply:
Thanks @joemcgill ! @ankitrox ☝️